lib/membrane_http_adaptive_stream/sink.ex

```diff
@@ -94,50 +95,34 @@ defmodule Membrane.HTTPAdaptiveStream.Sink do
     |> Map.merge(%{
       storage: Storage.new(options.storage),
       manifest: %Manifest{name: options.manifest_name, module: options.manifest_module},
-      awaiting_first_segment: MapSet.new(),
-      awaiting_discontinuities: %{}
+      awaiting_first_segment: MapSet.new()
     })
     ~> {:ok, &1}
   end
 
   @impl true
   def handle_caps(Pad.ref(:input, track_id), %CMAF.Track{} = caps, _ctx, state) do
-    {header_name, manifest, awaiting_discontinuity} =
+    {header_name, manifest} =
       if Manifest.has_track?(state.manifest, track_id) do
```

Comment:
I think that a comment explaining why we need to discontinue will be helpful for people that haven't touched the HLS plugin before.